src/commands/moderation/anti-raid-nuke.ts

```diff
@@ -123,7 +123,7 @@ export default class implements Command {
 						content: i18next.t('common.errors.timed_out', { lng: locale }),
 						components: [],
 					});
-				} catch (e) {
+				} catch (e: any) {
```

Comment:
I'd prefer if we don't `any` here and rather assert that it's a proper error class being thrown here.